blog/2025-07-cluster-api-provider-gardener.md

```diff
@@ -0,0 +1,129 @@
+| Target audience      | Infrastructure / platform engineers  | Application / service teams                |
+| Interface style      | Multiple manifests, component-driven | One manifest, opinionated, platform-driven |
+| Ownership            | End user manages infrastructure      | Platform team manages infrastructure       |
+| Multi-cloud support  | Configure it yourself                | Built-in                                   |
+| Lifecycle automation | Assemble from primitives             | Out-of-the-box automation                  |
+
+### Why did we build CAPGa?
+
+Gardener provides a powerful way to manage Kubernetes clusters at scale across many infrastructures. At the same time, Cluster API has become a widely adopted standard for cluster management. CAPGa aims to bridge these two projects, giving users a way to manage Gardener clusters with Cluster API-compatible controllers and tools.
+
+This unlocks several new use cases:
+- Users familiar with CAPI tooling can now manage Gardener-based clusters.
+- Integrators can plug Gardener into existing Cluster API-based workflows.
+- Platforms can gradually move towards a unified control plane managed by Gardener, while still using familiar CAPI resources.
+- The [multi-plane controller model](https://pages.github.tools.sap/ApeiroRA/apeirora-docu/next/best-practices/control-planes/crt) enables bi-directional interaction between Gardener and Cluster API, allowing users to manage clusters through both APIs seamlessly.
+
+### Key Features
+
+- **Cluster API compatibility**: Use familiar Cluster API resources to create and manage Kubernetes clusters.
+- **Still enjoy Gardener benefits**: Leverage Gardener’s existing support for various infrastructures, automatic version updates, hibernation and much more.
+- **KCP Support**: Integrated support for [KCP](kcp.io), the [New Era of Multi-Tenant Control Planes](https://documentation.apeirora.eu/blog/2025/03/25/kcp-multi-tenant-control-planes).
+
+### Demo
+
+![demo](./static/demo.gif)
+
+### Integrated KCP support
+
+CAPGa comes with built-in [KCP](https://www.kcp.io/) support, allowing users to interact with a shared, multi-tenant control-plane that is purpose-built for Kubernetes-like APIs beyond traditional container workloads.
+
+The long-term goal is to support CAPGa within a centrally managed _Platform Mesh_ built on top of KCP. This approach eliminates the need for users to provision and operate a dedicated Kubernetes cluster solely to host their Cluster API components. At the same time, users can interact directly with the Gardener API through the same unified control plane, enabling flexible and consistent cluster lifecycle management across both interfaces. This scenario is also illustrated in the image below.
+
+![Transition from managing Cluster API from a custom cluster to Platform Mesh](./static/capi-transition-custom-platform-mesh.svg)
```

Comment:
only the right hand side pic is needed.

also, graphically, make it the same CAPGa "various capi-provider/controller" box, with state.